warehouse/templates/includes/packaging/project-data.html

```diff
@@ -43,22 +43,40 @@ <h6>{% trans %}Maintainers{% endtrans %}</h6>
       </span>
     {% endfor %}
   {% endif %}
+  {% set verified_urls, unverified_urls = [],[] %}
+  {% for name, url in release.urls.items() %}
+    {% if is_valid_uri(url) %}
+      {% if name in ['Source', 'Homepage'] and project.is_verified_url(url) %}
+        {{ verified_urls.append((name, url)) }}
+      {% else %}
+        {{ unverified_urls.append((name, url)) }}
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+  {% if verified_urls.values() %}
+    <h6>{% trans %}Project links{% endtrans %}</h6>
+    {% for name, url in verified_urls %}
+    <li>
+      <a class="vertical-tabs__tab vertical-tabs__tab--with-icon vertical-tabs__tab--condensed" href="{{ url }}" rel="nofollow">
+        {{ url_icon(name, url) }}{{ name }} <i class="fa fa-circle-check check"></i>
+      </a>
+    </li>
+    {% endfor %}
+  {% endif %}
 </div>
 
 <div class="sidebar-section verified">
   <h3 class="sidebar-section__title">{% trans %}Unverified details{% endtrans %}</h3>
   <small><i>These details have <b>not</b> been verified by PyPI</i></small>
-{% if release.urls.values() | contains_valid_uris %}
+{% if unverified_urls.values() | contains_valid_uris %}
```

Comment:
We've already filtered valid URIs:

```suggestion
{% if unverified_urls.values() %}
```